doc/source/ray-overview/installation.rst

```diff
@@ -427,14 +427,14 @@ Docker images for Python 3.9.
     .. tab-item:: ray (Python 3.9)
         :sync: ray (Python 3.9)
 
-        Ray version: nightly (`e1d7025 <https://github.com/ray-project/ray/commit/e1d7025b4eb30911739681f33586be01f7dcbf5b>`_)
+        Ray version: nightly (`2eb4a81 <https://github.com/ray-project/ray/commit/2eb4a8119f903f79b78c01eaa9db06c6c390051c>`_)
```

Comment:
@khluu  this is not really from nightly..

Reply:
we can rename `nightly` with its SHA I think?

Reply:
this is just a doc section listing all dependencies up until the last time it's updated, so I think we can rename it as long as it's easy for user to understand

Reply:
picking from latest nightly build:
https://buildkite.com/ray-project/postmerge/builds/4047